From a98d92c059e0bd07cc08c74716b511e4dc3f460c Mon Sep 17 00:00:00 2001
From: nick87720z <nick87720z@gmail.com>
Date: Mon, 28 May 2018 14:07:52 +0500
Subject: [PATCH 1264/1936] fix path for save-osc test

Current version doesn't work if buildir not in root of source dir
---
 src/Tests/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Tests/CMakeLists.txt b/src/Tests/CMakeLists.txt
index 9ff1a640..9118dc7e 100644
--- a/src/Tests/CMakeLists.txt
+++ b/src/Tests/CMakeLists.txt
@@ -70,7 +70,7 @@ target_link_libraries(save-osc
                       zynaddsubfx_gui_bridge
                       ${GUI_LIBRARIES} ${NIO_LIBRARIES} ${AUDIO_LIBRARIES})
 #this will be replace with a for loop when the code will get more stable:
-add_test(SaveOsc save-osc ../../../instruments/examples/Arpeggio\ 1.xmz)
+add_test(SaveOsc save-osc ${CMAKE_CURRENT_SOURCE_DIR}/../../instruments/examples/Arpeggio\ 1.xmz)
 
 #message(STATUS "Plugin Test ${GUI_LIBRARIES} ${NIO_LIBRARIES} ${AUDIO_LIBRARIES}")
 
-- 
2.47.0

